crates/wash-cli/tests/wash_secrets.rs

```diff
@@ -52,26 +52,102 @@ async fn test_secret_put_and_get() -> anyhow::Result<()> {
         .get("type")
         .is_some_and(|v| v == SECRET_TYPE));
     let serde_json::Value::String(policy) = retrieved_secret
-        .get("policy_properties")
-        .context("policy_properties should exist")?
+        .get("policy")
+        .context("policy should exist")?
     else {
-        bail!("policy_properties should be a string");
+        bail!("policy should be a string");
     };
 
-    let policy: HashMap<String, String> =
-        serde_json::from_str(policy).context("policy_properties should be a JSON object")?;
-    assert!(policy.get("type").is_some_and(|t| t == SECRET_POLICY_TYPE));
-    let Some(properties) = policy.get("properties") else {
-        bail!("policy_properties should have a 'properties' field");
+    let serde_json::Value::Object(policy) =
+        serde_json::from_str(policy).context("policy should be a JSON object")?
+    else {
+        bail!("policy should be a JSON object");
     };
+    assert!(policy.get("type").is_some_and(
+        |t| *t == serde_json::Value::String(SECRET_POLICY_PROPERTIES_TYPE.to_string())
+    ));
+    let Some(properties) = policy.get("properties").cloned() else {
+        bail!("policy should have a 'properties' field");
+    };
+
     let policy_properties: HashMap<String, String> =
-        serde_json::from_str(properties).context("properties should be a JSON object")?;
-    assert_eq!(policy_properties.len(), 1);
-    assert!(policy_properties
-        .get("backend")
-        .is_some_and(|b| b == "baobun"));
+        serde_json::from_value(properties).context("properties should be a JSON object")?;
+    assert_eq!(policy_properties.len(), 0);
```

Comment:
```suggestion
    assert!(policy_properties.is_empty());
```

Surprised `clippy` said nothing!